apps/molgenis-components/src/components/forms/InputRefList.vue

```diff
@@ -234,7 +234,11 @@ export default {
         filter: this.filter,
         orderby: this.orderby,
       };
-      const response = await this.client.fetchTableData(this.tableId, options);
+      const response = await this.client.fetchTableData(
+        this.tableId,
+        options,
+        0
+      );
```

Comment:
This will break the labels of this input e.g. from the biobank input
![labels broken](https://github.com/molgenis/molgenis-emx2/assets/5484559/2cdb0420-75f5-49ab-8642-2850241ff961)
So is it really necessary to reduce it to 0? If so maybe we should try to find another solutions for this.

Reply:
Without this this screen still breaks https://preview-emx2-pr-3688.dev.molgenis.org/catalogue-demo/catalogue/#/variable-explorer

What fields are needed for this input? E.g. would it be enough to include all 'key=x' fields?
Then we would only need to change the expandLevel=0 to include all key fields.

Elegant solutions:
1a. Esther suggested to use a computed field as primary key that can have the label. Then you don't need to compute this at retrieval time.
1b. same as option 1 but then require this field to be a key=x. Then we could automatically change the expand to retrieve only the key fields to make sure it is included.
3. Alternatively, but takes more time to implement, is that we would include option to override the query in metadata (e.g. add graphql for the query by hand so you can precisely get what you want).

I think I would go for option (1b) because we could to this in this PR easily.

Reply:
I would say that we always get the ID and the label of the entity, but if the label is computed this should be moved to the database so we don't have the complexity in the frontend.

Reply:
ok. Then we will merge this and then think on a way to do the computed reflabel, either by hand (with current settings) or make it a platform feature.